examples/Dockerfile/centos.dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM dokken/centos-stream-9
+
+RUN yum -y install git g++ python3-pip curl zip unzip tar perl \
+    && pip install cmake
+
+WORKDIR /opt/tiledb
+COPY . .
+
+RUN cmake . -B build -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ -DCMAKE_MAKE_PROGRAM=make \
```

Comment:
I wonder, are these GCC/make explicit flags necessary?

Reply:
Well I don't think they hurt and they are good placeholders for when need to explicitly version to older or newer than default.

Reply:
For some unknown reason, the `cmake` command is not able to find them on Centos Stream9